ipam/calico-ipam.go

```diff
@@ -30,13 +30,19 @@ import (
 	"github.com/projectcalico/cni-plugin/utils"
 	"github.com/projectcalico/libcalico-go/lib/client"
 	"github.com/projectcalico/libcalico-go/lib/errors"
+	"github.com/projectcalico/libcalico-go/lib/logutils"
 	cnet "github.com/projectcalico/libcalico-go/lib/net"
 )
 
 // VERSION is filled out during the build process (using git describe output)
 var VERSION string
 
 func main() {
+	// Set up logging formatting.
+	log.SetFormatter(&logutils.Formatter{})
```

Comment:
will these config options also be applicable to the context loggers we have in this file? for example line 91, line 104, etc.? (same for other files)